### DISCUSSION TITLE
[Bug?]: Updated Docker container from 0.14-b2 to 0.14-b3, tempio command not found

### DISCUSSION THREAD

**Discussion by Calimerorulez** [blakeblackshear/frigate#11979] [Report a Bug] [answered]
### Describe the problem you are having

I just upgraded from 0.14-b2 to b3, now the container doesn't start:
```

BrokenPipeError: [Errno 32] Broken pipe
2024-06-15 21:11:59.720641027  [INFO] Service NGINX exited with code 127 (by signal 0)
2024-06-15 21:12:00.727915703  [INFO] Starting NGINX...
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
2024-06-15 21:12:00.792779915  [INFO] Service NGINX exited with code 127 (by signal 0)
2024-06-15 21:12:01.798199765  [INFO] Starting NGINX...
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
2024-06-15 21:12:01.861267391  [INFO] Service NGINX exited with code 127 (by signal 0)
```

### Steps to reproduce

Update from b2 to b3.

### Version

0.14-b3

### Frigate config file

```yaml
-
```

### Relevant log output

```shell
BrokenPipeError: [Errno 32] Broken pipe
2024-06-15 21:11:59.720641027  [INFO] Service NGINX exited with code 127 (by signal 0)
2024-06-15 21:12:00.727915703  [INFO] Starting NGINX...
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
2024-06-15 21:12:00.792779915  [INFO] Service NGINX exited with code 127 (by signal 0)
2024-06-15 21:12:01.798199765  [INFO] Starting NGINX...
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
2024-06-15 21:12:01.861267391  [INFO] Service NGINX exited with code 127 (by signal 0)
```

### Operating system

Proxmox

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Unifi

**Comment by NickM-27** [comment]
duplicate of https://github.com/blakeblackshear/frigate/discussions/11755 most likely

**Reply by Calimerorulez** [reply]
Ok thank you, I will try the solution mentioned. Using Portainer too here.

**Reply by Calimerorulez** [reply]
Recreating the container does not solve it in my case.

**Reply by NickM-27** [reply]
either way I don't think this is something for frigate to fix, this has only been reported on portainer and no one else and there's no reason tempio wouldn't be found unless something on the system was being cached incorrectly

**Reply by Calimerorulez** [reply]
I just added

`/usr/local/tempio/bin/`

to the path env variable in the Frigate container, and now it keeps running.

edit:
My complete PATH variable now is:

`/usr/lib/btbn-ffmpeg/bin:/usr/local/go2rtc/bin:/usr/local/nginx/sbin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/tempio/bin/`

**Reply by blakeblackshear** [reply, marked as answer]
Frigate's container already does this correctly: https://github.com/blakeblackshear/frigate/blob/dev/docker/main/Dockerfile#L156

Portainer is saving old PATH variables and overriding the PATH variable for updates. There is nothing we can do to prevent this from happening. File a bug report with Portainer.
